### PR TITLE
0.3.2 release notes

### PR DESCRIPTION
**Pull request by sofroniewn**
# Description
This PR adds release notes for 0.3.2. Note it doesn't include #1210 yet, if we get #1210 in time for the release I will add it, otherwise I think it is good to go

### CODE REVIEW

**Review by tlambert03 — APPROVED** [review]
yep, that's what happened! :)

File: docs/release/release_0_3_2.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+
+## New Features
+- General multithreading API and @thread_worker decorator (#1210)
+- Rich jupyter display for napari screenshots (#1269)
+
+## Improvements
+- make Qt component non private module (#1122)
+- add option to disable overwrite of labels during painting (#1264)
+- Refactor: move dask utils from misc.py to new dask_utils.py (#1265)
```
Comment (jni) [inline docs/release/release_0_3_2.md]: Generally I would reserve this section for user-facing improvements. Should we move this to an "other improvements" section?

**Review by jni — APPROVED** [review]
@sofroniewn I've made a very minor and optional comment, otherwise this is ready to go, short and sweet! =)

File: docs/release/release_0_3_2.md (hunk 1 above)
Comment (sofroniewn) [reply docs/release/release_0_3_2.md]: Let's save for next time, i did fix the capitolizations

**Pull request merged**